app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -415,8 +415,8 @@ class BrowserTabFragment :
     }
 
     private val autofillCallback = object : Callback {
-        override suspend fun onCredentialsAvailableToInject(credentials: List<LoginCredentials>) {
-            showAutofillDialogChooseCredentials(credentials)
+        override suspend fun onCredentialsAvailableToInject(credentials: List<LoginCredentials>, isAutoprompt: Boolean) {
+            showAutofillDialogChooseCredentials(credentials, isAutoprompt)
```

Comment:
I feel like I'm passing this prop around a bit too much, but couldn't find a better way.